src/components/Quiz/QuizzesModal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from "react"
+import {
+  Modal as ChakraModal,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalProps,
+} from "@chakra-ui/react"
+
+import { QuizzesHubContext } from "./context"
+
+interface IProps extends ModalProps {
+  children: React.ReactNode
+}
+
+const QuizzesModal: React.FC<IProps> = ({ children, ...rest }) => {
+  const { status: quizStatus } = useContext(QuizzesHubContext)
+
+  const statusColor =
+    quizStatus === "neutral"
+      ? "white"
+      : quizStatus === "success"
+      ? "successLight"
+      : "errorLight"
+
+  return (
+    <ChakraModal
+      isCentered
+      size={{ base: "full", md: "xl" }}
+      scrollBehavior="inside"
```

Comment:
For a better UX, I'd prefer the modal to not be closed if the user accidentally clicks outside the content (also the user will lose any progress made).

```suggestion
      scrollBehavior="inside"
      closeOnOverlayClick={false}
```

Reply:
I think it's better as it is now too as it's the common pattern and what users usually expect to happen (clen when clicking outside)

Reply:
Agree - clicking out to close is a pattern that we could respect here. There is the risk of losing your progress, but I think its minimal. @nhsz If I do 2 quizzes in a row and close the modal in the middle of the second one, will the previous quiz progress be saved?

Reply:
@konopkja no, partial progress is not saved, stats are only computed when you finish a quiz. Also partial progress wouldn't make sense as e.g.: you need to finish it to get the average

Reply:
resolved in DMs with Nico

Reply:
Sounds good!